hadoop-tools/hadoop-distcp/src/test/java/org/apache/hadoop/tools/TestExternalCall.java

```diff
@@ -66,7 +67,11 @@ private static Configuration getConf() {
   public void setup() {
 
     securityManager = System.getSecurityManager();
```

Comment:
`System.getSecurityManager()` will also be one Deprecate APIs based on JEP-411, Would you mind to process it together. Thanks.

Reply:
Thank you, I've moved it inside the try block, @Hexiaoqiao .